CHANGELOG.md

```diff
@@ -9,6 +9,20 @@ This release closes the [3.0.0 milestone](https://github.com/Instagram/IGListKit
 
 ### Breaking Changes
 
+- Renamed `withCollectionView:` param to `collectionView:` in `IGListAdapterUpdaterDelegate` [Malecks] (https://github.com/Malecks) [(#466)](https://github.com/Instagram/IGListKit/pull/466)
+
+```objc
+// OLD
+- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater 
+  didPerformBatchUpdates:(IGListBatchUpdateData *)updates 
+  withCollectionView:(UICollectionView *)collectionView;
+  
+// NEW
+- (void)listAdapterUpdater:(IGListAdapterUpdater *)listAdapterUpdater 
+  didPerformBatchUpdates:(IGListBatchUpdateData *)updates 
+  collectionView:(UICollectionView *)collectionView;
+```
```

Comment:
Mentioned this in the other PR, but probably don't need all this, I _think_ Xcode will offer the right suggestion.